README.md

```diff
@@ -61,52 +60,17 @@ The UI is done with FXML created with [Gluon's SceneBuilder](http://gluonhq.com/
 ### Q: Why Kotlin?  
 **A:** It has a lot of libraries (pick any random Java library...) It's pretty productive! Gee-whiz functional stuff baked in. Besides, if this project gets to a place where it's really awesome but just needs extra speed we can reach for that C++ or Rust book.
 
-### Q: Bedeutet "Kot" nicht 💩 in Deutsch?
```

Comment:
Why would you kill that? :D

Reply:
We definitely gotta leave it. It _really is_ a FAQ!